tmail-backend/integration-tests/jmap/distributed-jmap-integration-tests/src/test/java/com/linagora/tmail/james/DistributedLinagoraFilterStateChangeTest.java

```diff
@@ -35,6 +36,16 @@
         .extension(new AwsS3BlobStoreExtension())
         .server(configuration -> DistributedServer.createServer(configuration)
             .overrideWith(new TestJMAPServerModule())
-            .overrideWith(new FilterTypeNameModule()))
+            .overrideWith(new JmapGuiceFilterStateChangeModule()))
```

Comment:
We should not need any additional module here

Reply:
oops sorry i forgot

Reply:
Please **do not put your bindings here**. We need them in production.